Doc updates for My Access - new PR incorporating all feedback from previous reviews (#11361)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/accesspackageassignment-filterbycurrentuser.md

---

> title: "accessPackageAssignment: FilterByCurrentUser"

Reviewer: Same comment here - verify that this should be a lowercase f.

Author: done

---

> title: "accessPackageAssignment: s"

Reviewer 2: I think this should be: accessPackageAssignment: filterByCurrentUser

description: "Retrieve a list of accesspackageassignment objects filtered on the signed-in user."
localization_priority: Normal
author: "sbounouh"
ms.prod: "microsoft-identity-platform"
doc_type: "apiPageType"
---

# accessPackageAssignment: filterByCurrentUser
Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

In [Azure AD Entitlement Management](../resources/entitlementmanagement-root.md), retrieve a list of [accessPackageAssignment](../resources/accesspackageassignment.md) objects filtered on the signed-in user.

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|EntitlementManagement.Read.All, EntitlementManagement.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported.|
|Application|Not supported.|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /identityGovernance/entitlementManagement/accessPackageAssignments/filterByCurrentUser
```

> ## Query parameters

Reviewer 2: This should be: Function parameters

> The following table shows the parameters that can be used with this method.

Reviewer 2: ```suggestion
The following table shows the parameters that can be used with the function.
```

|Parameter|Type|Description|
|:---|:---|:---|

> |on|String|The list of options that can be used to filter on current user. Possible options are: `target`, `createdBy`. Additional options may be available in the future.|

Reviewer 2: ```suggestion
|on|String|The list of options that can be used to filter on current user. Possible values: `target`, `createdBy`.|
```

> - `target` is used to get the `accessPackageAssignment` objects where the signed-in user is the target. The resulting list includes all of the assignments, current and expired, that the caller has across all catalogs and access packages.

Reviewer 2: ```suggestion
- `target` is used to get the `accessPackageAssignment` objects where the signed-in user is the target. The resulting list includes all of the assignments, current and expired, for the caller across all catalogs and access packages.
```

> - `createdBy` is used to get the `accessPackageAssignment` objects created by the signed-in user. The resulting list includes all of the assignments that the caller has created for themselves or on behalf of others (such as in case of admin direct assignment), across all catalogs and access packages.

Reviewer 2: ```suggestion
- `createdBy` is used to get the `accessPackageAssignment` objects created by the signed-in user. The resulting list includes all of the assignments that the caller created for themselves or on behalf of others, such as in case of admin direct assignment, across all catalogs and access packages.
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body
Do not supply a request body for this method.

## Response

If successful, this method returns a `200 OK` response code and an [accessPackageAssignment](../resources/accesspackageassignment.md) collection in the response body.

> When a result set spans multiple pages, Microsoft Graph returns that page with an `@odata.nextLink` property in the response that contains a URL to the next page of results. If that property is present, continue making additional requests with the `@odata.nextLink` URL in each response, until all the results are returned. Refer to [paging Microsoft Graph data in your app](/graph/paging.md) for more details.

Reviewer 2: ```suggestion
When a result set spans multiple pages, Microsoft Graph returns that page with an `@odata.nextLink` property in the response that contains a URL to the next page of results. If that property is present, continue making additional requests with the `@odata.nextLink` URL in each response, until all the results are returned. For more information, see [paging Microsoft Graph data in your app](/graph/paging.md).
```
